tools/llm_bench/llm_bench_utils/ov_utils.py

```diff
@@ -279,6 +279,64 @@ def create_ldm_super_resolution_model(model_path, device, **kwargs):
     return ov_model, from_pretrained_time
 
 
+def create_genai_speech_2_txt_model(model_path, device, **kwargs):
+    import openvino_genai as ov_genai
+    if kwargs.get("genai", False) is False:
+        raise RuntimeError('==Failure the command line does not set --genai ==')
+    if is_genai_available(log_msg=True) is False:
+        raise RuntimeError('==Failure genai is not enable ==')
+    start = time.perf_counter()
+    genai_pipe = ov_genai.WhisperPipeline(model_path, device.upper())
+    end = time.perf_counter()
+    from_pretrained_time = end - start
+    log.info(f'From pretrained time: {from_pretrained_time:.2f}s')
+    processor = AutoProcessor.from_pretrained(model_path)
+    return genai_pipe, processor, from_pretrained_time, True
+
+
+def create_speech_2txt_model(model_path, device, **kwargs):
+    """Create speech generation model.
+
+    - model_path: can be model_path or IR path
+    - device: can be CPU
+    - model_type:
+    """
+    default_model_type = DEFAULT_MODEL_CLASSES[kwargs['use_case']]
+    model_type = kwargs.get('model_type', default_model_type)
+    model_class = OV_MODEL_CLASSES_MAPPING.get(model_type, OV_MODEL_CLASSES_MAPPING[default_model_type])
+    model_path = Path(model_path)
+    model_path_existed = model_path.exists()
+    # load model
+    if not model_path_existed:
+        raise RuntimeError(f'==Failure ==: model path:{model_path} does not exist')
+    else:
+        if kwargs.get("genai", False) and is_genai_available(log_msg=True):
+            if model_class not in [OV_MODEL_CLASSES_MAPPING[default_model_type]]:
+                log.warning("OpenVINO GenAI based benchmarking is not available for {model_type}. Will be switched to default bencmarking")
+            else:
+                return create_genai_speech_2_txt_model(model_path, device, **kwargs)
+        start = time.perf_counter()
+        ov_model = model_class.from_pretrained(
+            model_path,
+            device=device
+        )
+        end = time.perf_counter()
+    from_pretrained_time = end - start
+    log.info(f'From pretrained time: {from_pretrained_time:.2f}s')
+    processor = AutoProcessor.from_pretrained(model_path)
+
+    from transformers import pipeline
+    pipe = pipeline(
+        "automatic-speech-recognition",
+        model=ov_model,
+        chunk_length_s=30,
```

Comment:
`chunk_length_s` enables parallel chunking algorithm. It's intended to be used with `.generate(batch_size)`.
Parallel chunking algorithm is not implemented yet in genai. I propose to remove this parameter which will enable default sequential algorithm.

Reply:
updated